src/index.html

```diff
@@ -14,15 +14,15 @@ <h1>Leaderboard</h1>
       <section class="score-container">
         <div class="scores-header">
           <h2>Recent Scores</h2>
-          <button type="button">Refresh</button>
+          <button id="refreshBtn" type="button">Refresh</button>
         </div>
         <ul class="leaders-list" id="leadersList"></ul>
       </section>
       <section class="add-container">
         <h2>Add your score</h2>
         <form class="form-container">
-          <input type="text" />
-          <input type="text" />
+          <input id="userInput" type="text" />
+          <input id="scoreInput" type="text" />
```

Comment:
- [ ] [Optional] It would be better professionally if you make the type of the score input field as a number.